error C2039: 'bA�egin': is not a member of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'

```
D:\a\picguard\picguard\windows\flutter\ephemeral\.plugin_symlinks\file_saver\windows\file_saver_plugin.cpp(79,63): error C3872: '0xa6': this character is not allowed in an identifier [D:\a\picguard\picguard\build\windows\x64\plugins\file_saver\file_saver_plugin.vcxproj]
D:\a\picguard\picguard\windows\flutter\ephemeral\.plugin_symlinks\file_saver\windows\file_saver_plugin.cpp(79,63): error C2039: 'bA�egin': is not a member of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' [D:\a\picguard\picguard\build\windows\x64\plugins\file_saver\file_saver_plugin.vcxproj]
D:\a\picguard\picguard\windows\flutter\ephemeral\.plugin_symlinks\file_saver\windows\file_saver_plugin.cpp(79,48): error C2440: '<function-style-cast>': cannot convert from 'std::_String_iterator<std::_String_val<std::_Simple_types<_Elem>>>' to 'std::wstring' [D:\a\picguard\picguard\build\windows\x64\plugins\file_saver\file_saver_plugin.vcxproj]
D:\a\picguard\picguard\windows\flutter\ephemeral\.plugin_symlinks\file_saver\windows\file_saver_plugin.cpp(79,48): error C2440:         with [D:\a\picguard\picguard\build\windows\x64\plugins\file_saver\file_saver_plugin.vcxproj]
D:\a\picguard\picguard\windows\flutter\ephemeral\.plugin_symlinks\file_saver\windows\file_saver_plugin.cpp(79,48): error C2440:         [ [D:\a\picguard\picguard\build\windows\x64\plugins\file_saver\file_saver_plugin.vcxproj]
D:\a\picguard\picguard\windows\flutter\ephemeral\.plugin_symlinks\file_saver\windows\file_saver_plugin.cpp(79,48): error C2440:             _Elem=char [D:\a\picguard\picguard\build\windows\x64\plugins\file_saver\file_saver_plugin.vcxproj]
D:\a\picguard\picguard\windows\flutter\ephemeral\.plugin_symlinks\file_saver\windows\file_saver_plugin.cpp(79,48): error C2440:         ] [D:\a\picguard\picguard\build\windows\x64\plugins\file_saver\file_saver_plugin.vcxproj]
```

--- a/windows/file_saver_plugin.cpp
+++ b/windows/file_saver_plugin.cpp
@@ -76,7 +76,7 @@ std::wstring FileExtensionToFileFilter(std::string fileExtension) {
   std::string fileExtensionName = fileExtension.substr(1);
   for (auto& c : fileExtensionName) c = (char) std::toupper(c);
 
-  std::wstring wideFileExtension = std::wstring(fileExtension.bæegin(), fileExtension.end());
+  std::wstring wideFileExtension = std::wstring(fileExtension.begin(), fileExtension.end());
   std::wstring wideFileExtensionName = std::wstring(fileExtensionName.begin(), fileExtensionName.end());
   return wideFileExtensionName + L" File\0*." + wideFileExtensionName + L"\0\0";
 }

